modules/pekko-http/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Pekko-Http module for PureConfig
+
+Adds support for [Pekko-Http](https://pekko.apache.org/docs/pekko-http/current/common/http-model.html)'s Uri class to PureConfig. PRs adding support
+for other classes are welcome :)
+
+## Add pureconfig-pekko-http to your project
+
+In addition to [core PureConfig](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-pekko-http" % "@VERSION@"
+```
+
+## Example
+
+To load an `Uri` into a configuration, create a class to hold it:
+
+```scala mdoc:silent
+import pekko.http.scaladsl.model.Uri
```

Comment:
Our CI step that checks for mdoc output is broken.

```suggestion
import org.apache.pekko.http.scaladsl.model.Uri
```

Reply:
Oops, nice catch.
Fixed in 8655d5b